ui/app/templates/vault/cluster/access/identity/aliases/index.hbs

```diff
@@ -5,7 +5,7 @@
 
 <Identity::EntityNav @identityType={{this.identityType}} @model={{this.model}} />
 {{#if this.model.meta.total}}
-  {{#each this.model as |item|}}
+  {{#each (sort-objects this.model "name") as |item|}}
```

Comment:
I think we'll want to sort in `lazyPaginatedQuery` instead, otherwise this will only sort the items on the current page